pkg/template/pkg/resource_registry.go

```diff
@@ -14,6 +14,7 @@
 package pkg
 
 import (
+	"github.com/aws/aws-controllers-k8s/pkg/template"
```

Comment:
Please place this import below the standard imports, separated by an empty newline.

Reply:
Done.